packages/icons/bin/import/getSVGData.ts

```diff
@@ -0,0 +1,84 @@
+import * as Figma from 'figma-js'
+import { processFile } from 'figma-transformer'
+import axios from 'axios'
+
+const DEFAULT_BATCH_SIZE = 100
+
+type GetSVGDataConfig = {
+    apiToken: string
+    fileId: string
+    pageName: string
+    componentPrefix?: string
+    batchSize?: number
+}
+
+type SVGData = {
+    id: string
+    url: string
+    name: string
+}
+
+type SVG = SVGData & {
+    data: string
+}
+
+function extractComponentName (name: string): string {
```

Comment:
Please provide description of naming principles related to figma

Reply:
added jsdoc 👍